Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,58 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                string fullName = AskFullName();
+                int age = AskAge();
+                int weight = AskWeight();
+                double height = AskHeight();
+
+                Console.WriteLine($"{fullName} is {age} years old, his weight is {weight} kg and his height is {height:F1} cm.");
+
+                CalculateBmi(weight, height);
+            }
+        }
+
+        static string AskFullName()
+        {
+            Console.WriteLine("What is your first name?");
+            string firstname = Console.ReadLine();
+            Console.WriteLine("What is your family name?");
+            string surname = Console.ReadLine();
+
+            return firstname + " " + surname;
+        }
+
+        static int AskAge()
+        {
+            Console.WriteLine("What is your age?");
+            int age = Convert.ToInt32(Console.ReadLine());
+
+            return age;
+        }
+
+        static int AskWeight()
+        {
+            Console.WriteLine("How much do you weigh?");
+            int weight = Convert.ToInt32(Console.ReadLine());
+
+            return weight;
+        }
+
+        static double AskHeight()
+        {
+            Console.WriteLine("How tall are you?");
+            double height = Convert.ToDouble(Console.ReadLine());
+
+            return height;
+        }
+
+        static void CalculateBmi(int weight, double height)
```

Comment:
This is less 'CalculateBmi' and more 'ConsoleWriteBmi'. It would make more sense to return the calculated value and have the caller handler the write so that clean separation of responsibilities is maintained.

Reply:
Got it, naming variables & methods is def something i need to work on.